package.json

```
@@ -110,7 +110,7 @@
     "webpack-dev-server": "2.11.3"
   },
   "dependencies": {
-    "@okta/okta-auth-js": "1.17.0",
+    "@okta/okta-auth-js": "^2.3.1",
```

Question: should we start to use loose minor version or should we keep locked as before?

---

Just my two cents:

In the past, AuthJS and the Widget have been tightly coupled as we've exposed AuthJS properties directly onto the Widget. Because of this, any new feature in AuthJS's `tokenManager` object would **not** be  available to Widget consumers until the dependency version is updated. As a result, this allowed us to properly doc new features from AuthJS.

I'd suggest keeping us pinned to the latest major and minor versions, but be flexible on patches. That way, what we classify as a patch versions (hotfix, security, bugs, etc) do not require a new release of the Widget.

```suggestion
    "@okta/okta-auth-js": "~2.3.1",
```